<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EPINHYDW0313" ignored="0" name="Test" tests="4" failures="4" timestamp="12 Dec 2018 06:06:16 GMT" time="0.04" errors="0">
  <testcase name="statusCodeOfNewUserRegistrationTest" time="0.009" classname="com.epam.week3.rest_api_testing.RestAPIPostTest">
    <failure type="java.lang.AssertionError" message="expected [201] but found [0]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [0]
at com.epam.week3.rest_api_testing.RestAPIPostTest.statusCodeOfNewUserRegistrationTest(RestAPIPostTest.java:72)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- statusCodeOfNewUserRegistrationTest -->
  <testcase name="statusCodeOfOldUserRegistrationTest" time="0.001" classname="com.epam.week3.rest_api_testing.RestAPIPostTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [0]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [0]
at com.epam.week3.rest_api_testing.RestAPIPostTest.statusCodeOfOldUserRegistrationTest(RestAPIPostTest.java:90)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- statusCodeOfOldUserRegistrationTest -->
  <testcase name="successCodeOfNewUserRegistrationTest" time="0.0" classname="com.epam.week3.rest_api_testing.RestAPIPostTest">
    <failure type="java.lang.AssertionError" message="expected [OPERATION_SUCCESS] but found []">
      <![CDATA[java.lang.AssertionError: expected [OPERATION_SUCCESS] but found []
at com.epam.week3.rest_api_testing.RestAPIPostTest.successCodeOfNewUserRegistrationTest(RestAPIPostTest.java:81)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- successCodeOfNewUserRegistrationTest -->
  <testcase name="successCodeOfOldUserRegistrationTest" time="0.001" classname="com.epam.week3.rest_api_testing.RestAPIPostTest">
    <failure type="java.lang.AssertionError" message="expected [FAULT_USER_ALREADY_EXISTS] but found []">
      <![CDATA[java.lang.AssertionError: expected [FAULT_USER_ALREADY_EXISTS] but found []
at com.epam.week3.rest_api_testing.RestAPIPostTest.successCodeOfOldUserRegistrationTest(RestAPIPostTest.java:98)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- successCodeOfOldUserRegistrationTest -->
</testsuite> <!-- Test -->
